apps/wing/project-templates/wing/private-api/main.w

```diff
@@ -1,20 +1,84 @@
 bring cloud;
 bring http;
 
+/**
+ * The example below is a simple note-taking app.
+ * It uses a cloud.Bucket to store notes and a cloud.Api to expose a RESTful interface.
+ * 
+ * The api has two endpoints:
+ * - GET /note?name=NAME to get a note by name
+ * - PUT /note/:NAME to save a note by name
+ * 
+ * The app also includes two cloud.Functions to consume the api.
+ * - Consumer-PUT reads a string like: `NAME:NOTE` and calls the api to save it.
+ * - Consumer-GET reads a string like: `NAME` and calls the api to get the note.
+ * 
+ * These consumer functions are not required for the app to work, but since our api is private, they are useful for testing.
+ * As an example, if you deploy this app to AWS, you can use the AWS CLI to invoke these functions
+ * by running the following commands, you can test the api:
+ * `aws lambda invoke --cli-binary-format raw-in-base64-out --function-name <function-name> --payload "\"n1:this is my note\"" response.json`
+ * `cat response.json`
+ * 
+ * And retrieve the note:
+ * `aws lambda invoke --cli-binary-format raw-in-base64-out --function-name <function-name> --payload "\"n1\"" response.json`
+ * `cat response.json`
+ *
+ * If you have not made changes to this template you can find your function names in AWS using the following command:
+ * `aws lambda list-functions --query "Functions[?starts_with(FunctionName, 'Consumer')].FunctionName"`
+ */
```

Comment:
For me (as a non-patient non-native speaker) it seems like a lot of text, maybe it will be good to split it? lines 5-14 in this chunk, and lines 16-28 before the consumer functions
we don't add tests because they can't reach through the vpc?

Reply:
Stylistically I worry that moving large block of text like that into the middle of the code might make reading the actual code more cumbersome to the user who is trying to see how each component is used/invoked.

Ill add this to the list of questions to maybe run by users during workshop to see how they feel about the templates.